### PR TITLE
Launcher 2.0 feature needed changes (and a few misc changes)

### PR DESCRIPTION
**Pull request by hhyyrylainen**
**Brief Description of What This PR Does**
Thrive side changes needed for launcher 2.0

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: src/general/MainMenu.tscn [hunk 1]
```diff
@@ -199,58 +200,41 @@ __meta__ = {
...
-margin_bottom = 234.0
+margin_bottom = 194.0
 texture = ExtResource( 10 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
-[node name="Menus" type="Control" parent="MenuContainers"]
-anchor_top = 0.368
+[node name="Menus" type="CenterContainer" parent="MenuContainers"]
```
Comment (Kasterisk) [inline src/general/MainMenu.tscn:221]: This change is probably what's affecting the animation the most.

**Review by Kasterisk — COMMENTED** [review]
Main menu is getting too crowded with buttons, no problem for now but I think we'll need a major overhaul, eventually.

File: src/general/MainMenu.tscn (hunk 1 above)
Comment (hhyyrylainen) [reply src/general/MainMenu.tscn:221]: I found it to make more sense to make it this way as I didn't understand how it was previously even working...

Also this should hopefully be the last button ever needed in the main menu, at least I hope, which is why I added it to the main menu. If we ever need more buttons, then a different design will be needed...

Comment (Kasterisk) [reply src/general/MainMenu.tscn:221]: We both agree on a big overhaul then.

**Comment by hhyyrylainen** [comment]
This should now have everything that's absolutely necessary for 0.6.0 (which means that this is still missing the planned development feed that could be shown in the main menu).

**Pull request merged**